Blockzilla/BrowserViewController.swift

```diff
@@ -1658,6 +1658,9 @@ extension BrowserViewController: WebControllerDelegate {
         toggleURLBarBackground(isBright: !urlBar.isEditing)
         urlBar.progressBar.hideProgressBar()
         GleanMetrics.Browser.totalUriCount.add()
+        webViewController.evaluatePDFContentType {
+            if $0 { GleanMetrics.Browser.pdfViewerUsed.add() }
+        }
```

Comment:
if probably better to name the variables to know what you're actually checking here. `if $0` doesn't tell me much.

Reply:
Indeed. Thanks! :)